### PR TITLE
Update moment time format in Device and Gateway list page

### PR DESCRIPTION
**Pull request by sagar-patel-sls**
Shows "Never" when a timestamp is not available.

### CODE REVIEW

**Comment by brocaar** [comment]
Thanks, but this seems to be a lot more changes than just showing "Never" when the timestamp is not available.

**Comment by sagar-patel-sls** [comment]
I'm using a prettier formatter for ReactJS so it will format the whole file based on configuration.

**Comment by brocaar** [comment]
Would you mind reducing the PR to just showing the "Never" in case there is no timestamp?

I'm happy to discuss (automatic) formatting the source code and follow the best practices for JS / TSX code but please create a separate issue for this to start this discussion. If it turns out to be useful, then I propose to integrate this in the Makefile to make sure everybody uses the same formatting :-)

File: ui/src/views/devices/ListDevices.tsx [hunk 1]
```diff
@@ -69,18 +58,17 @@ class ListDevices extends Component<IProps, IState> {
           if (record.lastSeenAt !== undefined) {
             let ts = new Date(0);
             ts.setUTCSeconds(record.lastSeenAt.seconds);
-            return moment(ts).format("YYYY-MM-DD HH:mm:ss");
```
Comment (brocaar) [inline ui/src/views/devices/ListDevices.tsx:72]: I moved away from `fromNow()` on purpose. In some cases this shows something like 'in a few seconds' where it should have been 'a few seconds ago'. This happens when the user machine is not time-synced or when there is a minor time difference between the server and user machine.

**Comment by sagar-patel-sls** [comment]
> Would you mind reducing the PR to just showing the "Never" in case there is no timestamp?
okay 👍

**Pull request merged**